files/ipset_sync

```diff
@@ -36,6 +36,7 @@ function get_content_file() {
 
 function construct_ipset_dump() {
   local id=$1
+  local setname=${2:-$id}
```

Comment:
To be able to swap sets (in kernel space); I load the new content of the ipset in a new set called SWAP_origid and then swap origid and SWAP_origid (this works only if the headers match, but then it's atomic and instant)